### DISCUSSION TITLE
[logs] UI does not show correct log levels for source/destination logs

### DISCUSSION THREAD

**Discussion by cjwooo** [airbytehq/airbyte#33475] [Connector Questions]
Why does the Airbyte UI not display my source and destination logs with their specified log levels, e.g. warn, error, etc? They all seem to be at the “info” level, regardless of what level I put in the AirbyteLogMessage object. In older Airbyte versions, the logs shown in the UI would be prefixed with the log level. Now, they’re just prefixed with `source >` or `destination >`.

I’m running Airbyte both locally with docker and on k8s, with version 0.50.1 and 0.50.31 respectively.

**Comment by marcosmarxm** [comment]
Did you change any env variable to change the level of the logs?

**Reply by cjwooo** [reply]
The only log-level-related variable I see in our config is
```
# Although not present as an env var, expected by Log4J configuration.
LOG_LEVEL=INFO
```
which was from the example .env file.

**Comment by sajarin** [comment]
I'm not sure why it's not showing the correct level anymore. Do you happen to remember what prior version it was that was working?

**Reply by cjwooo** [reply]
I think it was <0.40.x

**Reply by cjwooo** [reply]
0.39.37

**Reply by sajarin** [reply]
Just one more annoying ask, do you have screenshots of the difference between the two UIs? I don't have an explainable reason for why this feature isn't present in the latest anymore, my best guess given that 0.39.37 is so old is that we removed it.

**Reply by cjwooo** [reply]
Actually, the version we used that shows the correct levels is even older: 0.33.8-alpha. 0.39.37-alpha has the newer version with the log levels.

0.33.8-alpha:
<img width="1045" alt="image" src="https://github.com/airbytehq/airbyte/assets/4491213/8b1c7681-7df2-419a-b869-b587922e1cd9">

0.50.31:
<img width="1110" alt="image" src="https://github.com/airbytehq/airbyte/assets/4491213/6fda3d25-23a5-4ce6-9c33-b256388c94b9">

**Reply by cjwooo** [reply]
I can understand simplifying the logging prefix to drop the `DefaultAirbyteStreamFactory` portion, but dropping the log level itself seems like a downgrade. Can it be put back?

**Reply by cjwooo** [reply]
bump @sajarin

**Reply by cjwooo** [reply]
bump @sajarin @marcosmarxm

[a comment by an automated account is not shown]
